src/context/build-services.js

```diff
@@ -1,29 +1,34 @@
 /* eslint-disable global-require */
+const createForestAdminClient = require('@forestadmin/forestadmin-client').default;
+
 module.exports = (context) =>
   context
     .addInstance('logger', () => require('../services/logger'))
+    .addUsingFunction('forestAdminClient', ({ env, logger }) => createForestAdminClient({
+      forestServerUrl: env.FOREST_URL,
+      envSecret: env.FOREST_ENV_SECRET,
+      logger: (level, ...args) => logger[level.toLowerCase()](...args),
```

Comment:
```suggestion
      logger: (level, ...args) => env.DEBUG? logger[level.toLowerCase()](...args) : () => {},
```

We don't want debugs info by default.